linux/step05_1_all_webapps.sh

```diff
@@ -0,0 +1,86 @@
+
+#!/bin/bash
+
+PY_ENV=${PY_ENV:-py27}
+
+# Figure details
+URL_FIGURE=http://downloads.openmicroscopy.org/figure/1.2.0/figure-1.2.0.zip
+NAME_FIGURE=figure-1.2.0
```

Comment:
I used the same approach for all apps i.e. URL name zip name mainly due to gallery /webtest